.github/workflows/bundleTest-thor.yml

```diff
@@ -0,0 +1,276 @@
+        type: boolean
+        description: 'Test core file generation'
+        required: false
+        default: false
+
+env:
+  ML_SUPPRESS_WARNING_FILES: "RegressionTestModified.ecl ClassificationTestModified.ecl"
+  ML_EXCLUDE_FILES: "--ef ClassicTestModified.ecl,SVCTest.ecl,ClassificationTestModified.ecl"
+  BUNDLES_TO_TEST: "ML_Core PBblas GLM  GNN DBSCAN LearningTrees TextVectors KMeans SupportVectorMachines LinearRegression LogisticRegression"       
+  uploadArtifact: false 
+  
+jobs:
+
+  main:
+    runs-on: ${{ inputs.os }}
+    steps:
+      - name: Free additional disk space (remove Android SDK + Tools)
+        run: |
+          sudo rm -rf /usr/local/lib/android
+
+      - name: Download Package
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.asset-name }}
+          path: ${{ inputs.asset-name }}
+
+      - name: Install Dependencies
+        shell: "bash"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+            git \
+            wget \
+            net-tools \
+            tzdata \
+            unzip \
+            xvfb \
+            libxi6 \
+            default-jdk \
+            gdb \
+            ${{ inputs.dependencies }}
+
+      - name: Install Package
+        shell: "bash"
+        run: |
+          sudo apt-get install -y -f ./${{ inputs.asset-name }}/*.deb
+          sudo chown -R $USER:$USER /opt/HPCCSystems
+          sudo xmlstarlet ed -L -u 'Environment/Software/ThorCluster/@slavesPerNode' -v 2 -u 'Environment/Software/ThorCluster/@channelsPerSlave' -v 1 /etc/HPCCSystems/environment.xml
+
+      - name: Install ML Dependencies
+        shell: "bash"
+        run: |  
+          sudo apt install libsvm-dev libsvm-tools 
+          sudo pip install tensorflow numpy keras
+          
+      - name: Start HPCC-Platform
+        shell: "bash"
+        run: |
+          export LANG="en_US.UTF-8"
+          ulimit -c 100
+          echo 'core_%e.%p' | sudo tee /proc/sys/kernel/core_pattern
+          sudo update-locale
+          sudo /etc/init.d/hpcc-init start
+          
+      - name: Core generation test
+        if: ${{ inputs.test-core-file-generation }}
+        shell: "bash"
+        run: |
+            echo """
+            boolean seg() := beginc++ #option action
+            #include <csignal>
+            #body
+            raise(SIGABRT);
+            return false;
+            endc++;
+            output(seg()); """ > crash.ecl
+            
+            ecl run -t hthor crash.ecl
+        continue-on-error: true
+
+      - name: Get test from Github
+        shell: "bash"
+        run: |
+          IFS=' ' read -a BUNDLES_TO_TEST <<< $BUNDLES_TO_TEST
+          BUNDLES_COUNT=${#BUNDLES_TO_TEST[@]}
+          for ((i=0; i<$BUNDLES_COUNT; i++))
+          do
+              BUNDLE_NAME=${BUNDLES_TO_TEST[i]}
+              BUNDLE_REPO="https://github.com/hpcc-systems/${BUNDLES_TO_TEST[i]}.git"
```

Comment:
Please output the bundle name and it's repo name as well (as you already do bellow with the bundle version)